src/OpenWeatherMapApi/OpenWeatherMapUtil.cs

```diff
@@ -0,0 +1,315 @@
+﻿using OpenWeatherMapApi.Responses;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace OpenWeatherMapApi
+{
+	/// <summary>
+	/// Utilities for OpenWeatherMap.
+	/// </summary>
+	class Util
+	{
+		/// <summary>
+		/// Converter to convert CurrentWeatherResponse's to string[]'s.
+		/// </summary>
+		public class Converter
+		{
+			/// <summary>
+			/// Converts your weather response to a string[] with all information. View the code for the order the array is built.
+			/// </summary>
+			/// <param name="weatherresponse">Your weather response</param>
+			/// <returns>string[]</returns>
+			public string[] Convert(CurrentWeatherResponse weatherresponse)
```

Comment:
Wouldn't this be better implemented as a ToString or ToStringArray function in the WeatherResponse object?